nodeup/pkg/model/protokube.go

```diff
@@ -310,6 +311,13 @@ func (t *ProtokubeBuilder) buildEnvFile() (*nodetasks.File, error) {
 		envVars["AZURE_STORAGE_ACCOUNT"] = os.Getenv("AZURE_STORAGE_ACCOUNT")
 	}
 
+	if t.CloudProvider == kops.CloudProviderScaleway {
+		envVars["SCW_ACCESS_KEY"] = os.Getenv("SCW_ACCESS_KEY")
+		envVars["SCW_SECRET_KEY"] = os.Getenv("SCW_SECRET_KEY")
+		envVars["SCW_DEFAULT_PROJECT_ID"] = os.Getenv("SCW_DEFAULT_PROJECT_ID")
+		// TODO: do we also need to pass the default region ?
```

Comment:
I would say yes. In particular it could also be useful to support all environment variables that the SDK supports https://github.com/scaleway/scaleway-sdk-go/tree/master/scw#environment-variables by doing so we can override all the settings of the SDK